instrumented-tests/http-server-mock/Tests/HTTPServerMockTests/HTTPServerMockTests.swift

```diff
@@ -77,7 +77,9 @@ final class HTTPServerMockTests: XCTestCase {
             )
         }
         let timeoutTime: TimeInterval = 2
-        let recordedRequests = try session.pullRecordedPOSTRequests(count: 2, timeout: timeoutTime)
+        let recordedRequests = try session.pullRecordedRequests(timeout: timeoutTime) { requests in
+            requests.count == 2
```

Comment:
there are many `count == N` in this PR, we can change them to `count >= N` to guard against any unexpected problem (eg: threading, double-counting, etc.)

Reply:
Indeed - updated for Logging and Tracing 👍. In RUM I'd rather not assert `session.viewVisits` by `>=`. It is critical to ensure that we get exactly the expected number of RUM Views, not more.

Also, it's not that convenient for RUM, as we have optional chaining in: `RUMSession.from(requests)?.viewVisits` so `==` works out of the box, but `>=` would require some default.